### PR TITLE
Add php 8.0 to the test matrix

### PR DESCRIPTION
**Pull request by gossi**
add it to the test workflow

### CODE REVIEW

**Comment by gossi** [comment]
I was doing this with github UI .... and it greatly failed 😅 Will try again from local machine

File: composer.json [hunk 1]
```diff
@@ -26,7 +26,7 @@
     "issues": "https://github.com/phootwork/phootwork/issues"
   },
   "require": {
-    "php": ">=7.2",
+    "php": ">=7.2 || 8.0.*",
```
Comment (peter279k) [inline composer.json:29]: This has been problematic version semver setting. And the Composer presents following errors:

```PHP
[UnexpectedValueException]                                                  
  Could not parse version constraint  8.0.*: Invalid version string " 8.0.*"  
```

Comment (peter279k) [reply composer.json:29]: How about setting `^8.0`?

**Comment by cristianoc72** [comment]
Closed via #62

**Pull request closed without merging**